scripts/hub-stress-test.py

```diff
@@ -405,9 +409,11 @@ def create_users(count, batch_size, endpoint, session, existing_users=[]):
     return users
 
 
-def start_server(username, endpoint, session):
+def start_server(username, endpoint, session, profile=None):
+    if profile:
+        profile = {"profile": profile}
```

Comment:
Yeah this looks correct, the docs aren't super clear on this they just say the request body gets passed through and shows up as `user_options` on the Spawner object which it does:

https://github.com/jupyterhub/jupyterhub/blob/69bb34b943162370ea8f585ee15d96feae3a8fed/jupyterhub/user.py#L552

And this is where that gets used on spawn in KubeSpawner:

https://github.com/jupyterhub/kubespawner/blob/0.14.1/kubespawner/spawner.py#L2130